src/Form/FieldCheckbox.js

```diff
@@ -1,4 +1,4 @@
-import classNames from 'classnames';
+import classNames from 'classnames/dedupe';
```

Comment:
with the addition of `classnames/dedupe` should we remove the `classNames` from `defaultProps` and place them in the `classNames()` functions within the component methods?